### PR TITLE
Dual Quaternion fix.

### PR DESCRIPTION
**Pull request by NissimHadar**
Not working yet.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7578/app_12516_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7578/HighFidelity-Beta-PR12516-00f3682bbee19058d6f39caa2d1390d36694362f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7578/HighFidelity-Beta-PR12516-00f3682bbee19058d6f39caa2d1390d36694362f.exe

File: libraries/render-utils/src/RenderPipelines.cpp [hunk 1]
```diff
@@ -568,6 +573,22 @@ void initForwardPipelines(ShapePlumber& plumber, const render::ShapePipeline::Ba
             Key::Builder().withMaterial().withSkinned().withTangents().withFade(),
             skinModelNormalMapFadeVertex, modelNormalMapFadePixel, batchSetter, itemSetter, nullptr, nullptr);
 
+    // Dual Quaternion
+    addPipeline(
+        Key::Builder().withMaterial().withSkinned(),
```
Comment (samcake) [inline libraries/render-utils/src/RenderPipelines.cpp]: these shapeKeys should also contain "withDualQuaternion"

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7585/app_12516_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7585/HighFidelity-Beta-PR12516-f52f146db5f70832d4ce6fb8c16beff7bcfc4012.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7585/HighFidelity-Beta-PR12516-f52f146db5f70832d4ce6fb8c16beff7bcfc4012.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7587/app_12516_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7587/HighFidelity-Beta-PR12516-7cae04883e85a747e6472049515a07b0093cc473.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7587/HighFidelity-Beta-PR12516-7cae04883e85a747e6472049515a07b0093cc473.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7591/app_12516_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7591/HighFidelity-Beta-PR12516-7ae37c21e07dd20e09f40138b8cbfee324b0ce45.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7591/HighFidelity-Beta-PR12516-7ae37c21e07dd20e09f40138b8cbfee324b0ce45.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7593/app_12516_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7593/HighFidelity-Beta-PR12516-faad930e352ecb28b5223457cb421a5487b778b8.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7593/HighFidelity-Beta-PR12516-faad930e352ecb28b5223457cb421a5487b778b8.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7599/app_12516_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7599/HighFidelity-Beta-PR12516-5d7e6651a761b0f96db5b9f81cb8f9cc437a3596.dmg

**Comment by AndrewMeadows** [comment]
These changes have been pulled into the more comprehensive PR-12530

#12530

**Pull request closed without merging**